src/GenVector.h

```diff
@@ -295,8 +295,12 @@ namespace gen {
     const std::size_t alloc_size = capacity * sizeof(limb_type*)
       + bitmap_size * sizeof(bitmap_uint);
     limb_type **new_start = static_cast<limb_type **>(operator new(alloc_size));
+#if __cplusplus == 201703L
+    std::uninitialized_copy(start, start+old_capacity, new_start);
+#else
     std::copy(start, start+old_capacity,
               std::raw_storage_iterator<limb_type**, limb_type*>(new_start));
+#endif
```

Comment:
Why not always use `std::uninitialized_copy`? It seems to exist since c++03. Ditto below